src/components/Routes/Routes.js

```diff
@@ -14,16 +28,23 @@ const Routes = () => (
                 exact
                 path={section.path}
                 component={(props) => (
-                    <section.component sectionKey={section.key} {...props} />
+                    <ContentWrapper fullscreen={section.info.fullscreen}>
+                        <section.component
+                            sectionKey={section.key}
+                            {...props}
+                        />
+                    </ContentWrapper>
```

Comment:
I dont Love what I had to do here. But previously `.contentWrapper` was round the routes, and I needed to support the fullscreen layout...